clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -119,15 +118,16 @@ lowerCirAttrAsValue(mlir::cir::ZeroAttr zeroAttr, mlir::Location loc,
 
 /// ConstStruct visitor.
 mlir::Value lowerCirAttrAsValue(mlir::cir::ConstStructAttr constStruct,
-                                mlir::Location loc,
+                                mlir::Location loc, mlir::ModuleOp module,
```

Comment:
May make sense to add `module` parameter to _all_ `lowerCirAttrAsValue` variants for coherence.

Reply:
Only worth if we're actually using the module. Can you move it to be the first param instead?

Reply:
Done.